src/App.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import './App.css';
-import Calculater from './Components/Calculater';
+import { Route, Switch } from 'react-router-dom';
+import Calculator from './pages/Calculator';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
+import Navbar from './pages/Navbar';
 
 class App extends React.PureComponent {
   render() {
     return (
-      <div className="App">
-        <h1>
-          Hello from App
-        </h1>
-        <Calculater />
+      <>
+        <Navbar />
+        <Switch>
+          <Route exact path="/">
+            <div className="App">
+              <Home />
 
-      </div>
+            </div>
+          </Route>
+          <Route exact path="/calculator">
+            <Calculator />
+          </Route>
+          <Route exact path="/quote">
+            <Quote />
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", Please make the quote on the `Quote` page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md), the quote is centered aligned not left aligned.  Please kindly fix that.